prow/pjutil/pjutil.go

```diff
@@ -42,14 +42,15 @@ func NewProwJob(spec prowapi.ProwJobSpec, extraLabels, extraAnnotations map[stri
 	labels, annotations := decorate.LabelsAndAnnotationsForSpec(spec, extraLabels, extraAnnotations)
 	specCopy := spec.DeepCopy()
 	setReportDefault(specCopy)
+	uuidV1, _ := uuid.NewV1()
 
 	return prowapi.ProwJob{
 		TypeMeta: metav1.TypeMeta{
 			APIVersion: "prow.k8s.io/v1",
 			Kind:       "ProwJob",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        uuid.NewV1().String(),
+			Name:        uuidV1.String(),
```

Comment:
/assign @cjwagner

Reply:
Changed to using `google/uuid` as suggested, and diff is even smaller `uuid.NewV1().String()` => `uuid.New().String()`

Reply:
And reworked to be V1, and not V4, so diff is back.